glide.lock

```diff
@@ -0,0 +1,131 @@
+hash: df48123f9cd400234264795207b4dca1f90b46ee3b919e5162c1cbc057e93da4
```

Comment:
Looks like this slipped in accidentally. `glide.lock` should have been removed during the upgrade to `dep`. You'll have to remove it here.